scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -69,7 +69,7 @@ task AnnotateIntervals {
 
     runtime {
         docker: "${gatk_docker}"
-        memory: select_first([mem, 5]) + " GB"
+        memory: select_first([mem_gb, 5]) + " GB"
```

Comment:
This is strange because if `mem_gb` is supplied the machine memory and command memory are the same, but if it's not supplied they use different defaults.

Reply:
Oops, good catch.  I think for some of these more minor tasks we didn't do the whole machine_mem/command_mem thing.  I'll go back and standardize.